### PR TITLE
Improve testing script

### PR DESCRIPTION
**Pull request by polskikiel**
**Description**

Changes proposed in this pull request:

Improved testing script:
- Increased the frequency of calling for TestSuite status.
- `Waiting...` log is still displayed only once per minute

### CODE REVIEW

File: installation/scripts/testing.sh [hunk 1]
```diff
@@ -26,12 +26,15 @@ metadata:
   name: ${suiteName}
 spec:
   maxRetries: 1
-  concurrency: 1
+  concurrency: 2
```
Comment (aszecowka) [inline installation/scripts/testing.sh]: no, no, no - we have to know which tests can be run concurrently and only after that we can enable concurrency

File: installation/scripts/testing.sh [hunk 2]
```diff
@@ -26,12 +26,15 @@ metadata:
...
 EOF
 
 startTime=$(date +%s)
 
 testExitCode=0
+minCache=-1
```
Comment (aszecowka) [inline installation/scripts/testing.sh]: minCache is a very enigmatic name for me

File: installation/scripts/testing.sh [hunk 3]
```diff
@@ -40,6 +43,23 @@ do
     statusFailed=$(${kc} get cts ${suiteName}  -ojsonpath="{.status.conditions[?(@.type=='Failed')]}")
     statusError=$(${kc} get cts  ${suiteName} -ojsonpath="{.status.conditions[?(@.type=='Error')]}" )
 
+    succeededTestsNames=$(${kc} get cts ${suiteName} -ojsonpath="{.status.results[?(@.status=='Succeeded')].name}")
+    failedTestsNames=$(${kc} get cts ${suiteName} -ojsonpath="{.status.results[?(@.status=='Failed')].name}")
+
+    names=(${succeededTestsNames})
+    len="${#names[@]}"
+    if ((${succeededTests} < ${len})); then
```
Comment (aszecowka) [inline installation/scripts/testing.sh]: I think that it won't work if 2 tests finished at the same time. You will print only the name of the last one.

File: installation/scripts/testing.sh [hunk 4]
```diff
@@ -56,15 +76,28 @@ do
...
     sec=$((currTime-startTime))
     min=$((sec/60))
     if (( min > 60 )); then
         echo "Timeout for test suite '${suiteName}' occurred."
         testExitCode=1
         break
     fi
-    echo "ClusterTestSuite not finished. Waiting..."
-    sleep 60
+    if (( $minCache != $min )); then
```
Comment (aszecowka) [inline installation/scripts/testing.sh]: I am afraid that this code becomes too complicated. Maybe you can focus right now on improving script by more frequent checks if a suite is finished?

**Comment by aszecowka** [comment]
It looks that there can be an error in logic because I got the following output:
```
Test 'cluster-users-test' succeeded
Test 'cluster-users-test' succeeded
```

File: installation/scripts/testing.sh (hunk 4 above)
Comment (polskikiel) [reply installation/scripts/testing.sh]: discussed

File: installation/scripts/testing.sh (hunk 2 above)
Comment (polskikiel) [reply installation/scripts/testing.sh]: changed to `previousPrintTime`

**Pull request merged**